qa/vagrant/build.gradle

```diff
@@ -102,30 +102,26 @@ subprojects { Project platformProject ->
 
 }
 
-configurations {
-  allPlugins
-}
-
-List<String> plugins = []
+List<String> pluginNames = []
 for (Project subproj : project.rootProject.subprojects) {
   if (subproj.parent.path == ':plugins' || subproj.path.equals(':example-plugins:custom-settings')) {
     // add plugin as a dep
     dependencies {
-      allPlugins project(path: "${subproj.path}", configuration: 'zip')
+      packaging_plugins project(path: "${subproj.path}", configuration: 'zip')
     }
-    plugins.add(subproj.name)
+    pluginNames.add(subproj.name)
   }
 }
-plugins = plugins.toSorted()
+pluginNames = pluginNames.toSorted()
 
-copyPackagingArchives {
-  from configurations.allPlugins
+copyPlugins {
+  inputs.property("plugins", pluginNames)
```

Comment:
Could we simply _infer_ all this nonsense from all the dependencies added to the plugins configuration?

Reply:
I removed the inputs, since that is inferred by the configuration. But we need to still write the file, since that is currently required by the bats upgrade tests.